opensearch-mixin/.lint

```diff
@@ -0,0 +1,15 @@
+exclusions:
+  template-job-rule:
+    reason: "Prometheus datasource variable is being named as prometheus_datasource now while linter expects 'datasource'"
+  panel-datasource-rule:
+    reason: "Loki datasource variable is being named as loki_datasource now while linter expects 'datasource'"
+  template-datasource-rule:
+    reason: "Based on new convention we are using variable names prometheus_datasource and loki_datasource where as linter expects 'datasource'"
+  template-instance-rule:
+    reason: "Based on new convention we are using variable names prometheus_datasource and loki_datasource where as linter expects 'datasource'"
+  target-instance-rule:
+    reason: "The dashboard monitors on a cluster, node or index level, not an instance level."
+    entries:
+    - dashboard: "OpenSearch cluster overview"
+    - dashboard: "OpenSearch node overview"
+    - dashboard: "OpenSearch search and index overview"
```

Comment:
```suggestion
    - dashboard: "OpenSearch search and index overview"
  panel-units-rule:
    reason: "Custom units are used for better user experience in these panels"
    entries:
    - panel: (panel name)
```

run `make fmt` and then `make lint` which will give errors for panels that have custom units or no units. 
Should add each panel that this affects under `entries`